Resources/Prototypes/Entities/Objects/Misc/fire_extinguisher.yml

```
@@ -84,3 +84,36 @@
     animationState: extinguish
+
+- type: entity
+  name: pocket fire extinguisher
+  parent: FireExtinguisher
+  id: FireExtinguisherMini
+  description: A light and compact fibreglass-framed model fire extinguisher. It holds less water then its bigger brother.
+  components:
+  - type: Sprite
+    sprite: Objects/Misc/fire_extinguisher_mini.rsi
+    layers:
+        - state: fire_extinguisher_closed
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 8

(indentation)